doc/sphinx-guides/source/container/dev-usage.rst

```diff
@@ -0,0 +1,49 @@
+Development Usage
+=================
+
+Please note! This Docker setup is not for production!
+
+.. contents:: |toctitle|
+        :local:
+
+Intro
+-----
+
+Assuming you have `Docker <https://docs.docker.com/engine/install/>`_, `Docker Desktop <https://www.docker.com/products/docker-desktop/>`_,
```

Comment:
Could we just link to the OCI container spec (https://opencontainers.org/) without endorsing one implementation?

Reply:
No we can't. We can add Rancher Desktop if you want. Other than that, the Docker Maven Plugin *requires* using some sort of Moby daemon (Docker is a supported, paid distribution of Moby). You will not be able to built an image with podman or JIB etc with the current configuration.

It also is important to keep in mind that a future vision is to use these images with Testcontainers for more extensive tests than available now. This will also require some kind of Docker/Moby.

Reply:
It would be nice to habe this information in the documentation.

Reply:
@johannes-darms sounds fine but this PR is in "Ready for QA" and it would be a shame to lose momentum. If you can make a PR into this PR, I'm happy to look and hopefully merge.